arm/Microsoft.EventHub/namespaces/readme.md

```diff
@@ -306,6 +306,7 @@ userAssignedIdentities: {
 module namespaces './Microsoft.EventHub/namespaces/deploy.bicep' = {
   name: '${uniqueString(deployment().name)}-namespaces'
   params: {
+  
```

Comment:
Remove blank line

```suggestion
```

Reply:
@MariusStorhaug I accepted these last time, but this will fail the pipeline. The autoformatter expects an empty line here.